### ISSUE TITLE
Improved Support for Visonic MCT-340 E Door/Window Sensor

### ISSUE THREAD

**Issue by meremortals70** [dresden-elektronik/deconz-rest-plugin#3361]
I know there is some support for this sensor however I don't get all the sensors in Phoscon or HA. Are we able to improve on the current support please?

![image](https://user-images.githubusercontent.com/52735978/95047354-405fb600-0729-11eb-8f64-742ef75ac884.png)

## Device

- Product name: Visonic MCT-340 E
- Manufacturer: Visonic
- Model identifier: MCT-340e
- Device type : Sensor

## Screenshots

![image](https://user-images.githubusercontent.com/52735978/95047526-92084080-0729-11eb-995a-f11efa83232e.png)

![image](https://user-images.githubusercontent.com/52735978/95047572-a8ae9780-0729-11eb-95fd-fc3c50304ddd.png)

![image](https://user-images.githubusercontent.com/52735978/95047627-c4b23900-0729-11eb-8ba3-2f1952424990.png)

![image](https://user-images.githubusercontent.com/52735978/95047696-deec1700-0729-11eb-8703-12c1be907cd2.png)

![image](https://user-images.githubusercontent.com/52735978/95047757-fa572200-0729-11eb-8fb8-7305c2aceedf.png)

![image](https://user-images.githubusercontent.com/52735978/95047795-0e9b1f00-072a-11eb-9de2-f98fd8d2e0b2.png)

![image](https://user-images.githubusercontent.com/52735978/95047895-35f1ec00-072a-11eb-8a60-6fe36521031f.png)

![image](https://user-images.githubusercontent.com/52735978/95047998-65085d80-072a-11eb-893f-cf3cf44d1644.png)

![image](https://user-images.githubusercontent.com/52735978/95048073-88cba380-072a-11eb-9432-3908915b108e.png)

**Comment by mamrai1** [comment]
Yes, also for the Bosch motion sensor only motion and temp are shown in HA.. Battery state is shown only in phoscon.. Tamper sensor missing in both
